hazelcast/src/main/java/com/hazelcast/jet/pipeline/JdbcPropertyKeys.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.pipeline;
+
+import com.hazelcast.function.FunctionEx;
+
+import java.util.Properties;
+
+public final class JdbcPropertyKeys {
+
+    /**
+     * Property key to be passed to {@link Sources#jdbc(String, String, Properties, FunctionEx)}
```

Comment:
It should describe what it changes in the connector, usage intention can be added as `@see Sources#jdbc`

Reply:
rev : 3ffc869cd2313bfd7ee37d11781f3d4fd1152953